custom-bundle.Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o csv-merger csv-merger.g
 #ARG KEYSTONE_BUNDLE=quay.io/openstack-k8s-operators/keystone-operator-bundle:latest
 FROM quay.io/openstack-k8s-operators/keystone-operator-bundle:latest as keystone-bundle
 FROM quay.io/openstack-k8s-operators/mariadb-operator-bundle:latest as mariadb-bundle
+FROM quay.io/dprince/rabbitmq-cluster-operator-bundle:latest as rabbitmq-bundle
```

Comment:
Should we push this image to `quay.io/openstack-k8s-operators` instead?

Reply:
Yes. Will correct it.

Reply:
Just pushed. Actually had to fix issues with the new bundle build first. The latest build shows as master-latest instead of 'latest' which seems fine but may need to follow up on this to make sure it tracks correctly. Not sure why other repos are using just 'latest'